strongbox-user-management/src/main/resources/etc/conf/security-users.xml

```diff
@@ -27,7 +27,66 @@
             <encryption-algorithm>plain</encryption-algorithm>
         </credentials>
         <roles>
-            <role>USER_ROLE</role>
+            <role>ADMIN</role>
         </roles>
-    </user>    
+    </user>
+    <user>
```

Comment:
@AlexOreshkevich ,

I know you knocked this up quickly for the sake of testing, but would it be possible to add a method to one of the base test classes, so that these can be configured programatically during tests? I mean -- we wouldn't like to be leaking test resources (which, in effect, this currently is) to the public. Clearly, there will be no `storage0:act-releases-1` outside of our tests. :)